src/less/nav.less

```diff
@@ -245,3 +236,25 @@
 }
 
 }
+
+global-menu-container {
```

Comment:
@pascualin what's the purpose of this? the container is overlaying the whole screen and making all links, buttons, etc. inaccessible.

![image](https://user-images.githubusercontent.com/130654/26941523-a03a31fe-4c33-11e7-89e9-fcdd3fad41b1.png)